docs/testing/platform-limits.md

```diff
@@ -0,0 +1,44 @@
+---
+title: 'Platform limits with Office Scripts'
+description: 'Resource limits and browser support for Office Scripts when used with Excel on the web'
+ms.date: 07/23/2020
+localization_priority: Normal
+---
+
+# Platform limits with Office Scripts
+
+There are some platform limitations of which you be aware when developing Office Scripts. This article details browser support and data limits.
+
+## Browser support
+
+Office Scripts work in any browser that [supports Office for the web](https://support.microsoft.com/office/ad1303e0-a318-47aa-b409-d3a5eb44e452). However, some JavaScript features aren't supported in Internet Explorer 11 (IE 11). Any features introduced in [ES6 or later](https://www.w3schools.com/Js/js_es6.asp) won't work with IE 11. If people in your organization still use that browser, be sure to test your scripts in that environment when sharing them.
+
+### Third-party cookies
+
+Your browser needs third-party cookies enabled to show the **Automate** tab in Excel on the web. Check your browser settings if the tab isn't being displayed. If you're using a private browser session, you may need to re-enable this setting each time.
```

Comment:
Should we add image? I know images can go stale... 
In the private mode, user should allow "all cookies" (for Chrome). Edge may use its own terminology..

Reply:
I'm hesitant to add an image because we'd need to address several browsers. Considering this doesn't seem common, I'm hoping people can be successful using search for their browser. If you hear of any problems on Stack or other sources, please let me know and we can enhance this section.